assets/src/infrastructure/ui/inputs/dateTime/DateTimeRangePicker/style.scss

```diff
@@ -0,0 +1,11 @@
+.ee-date-time-range-picker {
+	.react-datetimerange-picker {
+		&__inputGroup {
+			&__month {
+				option {
+					display: none;
+				}
+			}
+		}
+	}
+}
```

Comment:
maybe we should use the css tricks for hiding content except for screen readers, like:

```css
      .sr-only {
            position: absolute;
            width: 1px;
            height: 1px;
            padding: 0;
            margin: -1px;
            overflow: hidden;
            clip: rect(0, 0, 0, 0);
            white-space: nowrap; /* added line */
            border: 0;
      }
```

Reply:
@tn3rb I've tried the approach you've posted above and the latest bootstrap mixin to visually hidden, but got no luck;
also, tried to find some insights on stackoverflow and such and it doesn't seem smth straightforward to deal with in this case of `select` element and the `a11y` implications :disappointed: 
since we don't have much control through the API, an ultimate solution might be to clone and adapt `react-datetimerange-picker`, but that might fall outside of the scope of this PR